perf-tool/src/agent.cpp

```diff
@@ -123,6 +124,16 @@ void parseAgentOptions(const char *options)
         }
     }
 
+//check if commandspath exists or not
+std::ifstream ifile;
+ifile.open(commandsPath);
+if(!ifile)
+{
+     printf("commandspath: %s\n", commandsPath.c_str());
+     printf("doesn't exists\n");
+     exit(0);
+}
+
```

Comment:
Since in OpenJ9 we avoid using streams, let's use the traditional C way of determining whether the file exists (fopen/fclose)

Reply:
Also, since this is an usage error, maybe we should print the message to stderr.

Reply:
Also, you do not need two printf statements. One is enough. 
Typo: "doesn't exists" --> "doesn't exist"